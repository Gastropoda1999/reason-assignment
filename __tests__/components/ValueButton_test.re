open Jest;
open Expect;

let _ =

describe "ValueButton" (fun _ => {
  test "renders" (fun _ => {
    let dispatch = fun _ => ();
    let value = "1";
    let tree = ValueButton.createElement ::dispatch ::value children::[] ()
      |> Renderer.render;

    expect tree |> toMatchSnapshot;
  });

  test "renders expanded button" (fun _ => {
    let dispatch = fun _ => ();
    let expand = true;
    let value = "1";
    let tree = ValueButton.createElement ::dispatch ::expand ::value children::[] ()
      |> Renderer.render;

    expect tree |> toMatchSnapshot;
  });
});
