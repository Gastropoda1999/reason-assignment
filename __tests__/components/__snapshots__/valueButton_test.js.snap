// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValueButton renders 1`] = `
<button
  className="css-1nylyli"
  onClick={[Function]}
  value="1"
>
  1
</button>
`;

exports[`ValueButton renders expanded button 1`] = `
<button
  className="css-1nylyli css-1x9fsos"
  onClick={[Function]}
  value="1"
>
  1
</button>
`;
